packages/data/readme.md

```diff
@@ -81,3 +81,18 @@ await data.query({
 	]
 })
 ```
+
+## Dependencies
+This shows how the packages depend on each other. 
+
+```mermaid
+flowchart LR
+    api --> sql-adapter
+	sql-adapter ---> db1[(datastore)]
+	sql-adapter  --> data-sql
+	sql-adapter --> data
+	api --> data
+    api --> no-sql-adapter
+	no-sql-adapter --> data
+	no-sql-adapter ---> db2[(datastore)]
+```
```

Comment:
[Here](https://mermaid.live/edit#pako:eNplULEOgjAQ_RVykyZ00JHByVEX3bQOJz2EBNpajhhD-HcPiBGkU997l_feXQupMwQJZKV7pTkGjg4nbSN56ItIqV1UP0uFBj1T0FbzBIosurlvriuDjDW7QOvb_9Bg0utK2KXDqPX8N3DE0wrWqb8Wc2Zms5CGjttZR4iholBhYWT1tg_TwDlVpCGRr6EMm5I1aNvJKDbszm-bQsKhoRgaL1a0L_ARsIIkw7IW1qO9OPfDZAoJO47nHa7cfQBROHvY) is a live version of the diagram above. For me such visualization makes it way faster to understand.

Reply:
I agree! ..Although we do gotta make sure the chart is accurate then 😉

Reply:
you mean regarding maintaining in the long run? or is something currently wrong with it? 🤔

Reply:
The current version is inaccurate! The API wouldn't talk to the drivers directly, but instead would go through `data` 👍🏻

Reply:
Yeah, I think it should look something like this:
```mermaid
graph LR;
    api --> data
    data --> sql-adapter
    data --> no-sql-adapter
	sql-adapter ---> db1[(datastore)]
	sql-adapter  --- data-sql
	no-sql-adapter ---> db2[(datastore)]
```

Reply:
That's correct!